trollmoves/server.py

```diff
@@ -292,7 +292,16 @@ def _run(self):
                 try:
                     address, _, payload = multiparts
                 except ValueError:
-                    LOGGER.warning("Invalid request, ignoring.")
+                    LOGGER.warning("Invalid request.")
+                    try:
+                        address = multiparts[0]
+                    except (TypeError, IndexError):
+                        LOGGER.warning("Address unknow, not sending an error message back.")
+                    else:
+                        message = Message('error', 'error', "Invalid message recieved")
+                        Thread(target=self.reply_and_send,
+                               args=(self.unknown, address, message)).start()
```

Comment:
Does this thread terminate by itself?

Reply:
`self.unknown` isn't defined in the `RequestManager` class. Is it something inherited from the `Thread` class?

Reply:
Nevermind, `self.unknown` is already there in the current `main` branch. But why does this show here in the diff as an addition?

Reply:
yes, the thread sends the answer and dies